src/Mono.Android/Android.Runtime/InputStreamInvoker.cs

```diff
@@ -6,13 +6,18 @@ namespace Android.Runtime
 	public class InputStreamInvoker : Stream
 	{
 		public Java.IO.InputStream BaseInputStream {get; private set;}
+		public Java.Nio.Channels.FileChannel BaseFileChannel {get; private set;}
 
 		public InputStreamInvoker (Java.IO.InputStream stream)
 		{
 			if (stream == null)
 				throw new ArgumentNullException (nameof (stream));
 
-			this.BaseInputStream = stream;
+			BaseInputStream = stream;
+
+			Java.IO.FileInputStream fileStream = stream as Java.IO.FileInputStream;
+			if (fileStream is object)
```

Comment:
Why not `fileStream != null`?

Reply:
Just a preference (https://twitter.com/jaredpar/status/1115019017297596416), if you like me to change it no problem either.